pkg/networkmanager/firewall-client.js

```diff
@@ -77,15 +77,17 @@ function initFirewalldDbus() {
         firewall.services = {};
         firewall.enabledServices = new Set();
 
+        console.info('initFirewalldDbus', JSON.stringify({ firewall, event, owner }));
+
         if (!firewall.enabled) {
             firewall.dispatchEvent('changed');
             return;
         }
 
         getZones()
                 .then(() => getServices())
-                .then(() => firewall.debouncedEvent('changed'))
-                .catch(error => console.warn(error));
+                .then(() => { console.info('debouncedEvent changed'); firewall.debouncedEvent('changed') })
```

Comment:
This should probably be `return firewall.debounced...` now it always returns `undefined`.

Reply:
That was a red herring, debouncedEvent doesn't return anything important.